commerce_coordinator/apps/commercetools/sub_messages/tasks.py

```diff
@@ -155,39 +161,57 @@ def fulfill_order_placed_message_signal_task(
         ct_program_product = client.get_product_by_program_id(bundle_id) if bundle_id else None
 
         product_title = ct_program_product.name.get('en-US', '') if ct_program_product else item.name.get('en-US', '')
-        serializer = OrderFulfillViewInputSerializer(data={
-            **default_params,
-            # Due to CT Variant SKU storing different values for course and entitlement models
-            # For bundle purchases, the course_id is the course_uuid
-            # For non-bundles purchase, the course_id is the course_run_key
-            'course_id': get_edx_product_course_run_key(item),
-            'line_item_id': item.id,
-            'course_mode': get_course_mode_from_ct_order(item),
-            'item_quantity': item.quantity,
-            'line_item_state_id': line_item_state_id,
-            'message_id': message_id,
-            'user_first_name': customer.first_name,
-            'user_last_name': customer.last_name,
-            'user_email': customer.email,
-            'product_title': product_title,
-            'product_type': item.product_type.obj.key
-        })
-
-        # the following throws and thus doesn't need to be a conditional
-        serializer.is_valid(raise_exception=True)  # pragma no cover
-
-        payload = serializer.validated_data
-
-        if bundle_id:
-            fulfill_order_placed_send_entitlement_signal.send_robust(
-                sender=fulfill_order_placed_message_signal_task,
-                **payload
-            )
+
+        serializer_data = {
+                **default_params,
+                # Due to CT Variant SKU storing different values for course and entitlement models
+                # For bundle purchases, the course_id is the course_uuid
+                # For non-bundles purchase, the course_id is the course_run_key
+                'course_id': get_edx_product_course_run_key(item),
+                'line_item_id': item.id,
+                'course_mode': get_course_mode_from_ct_order(item),
+                'item_quantity': item.quantity,
+                'line_item_state_id': line_item_state_id,
+                'message_id': message_id,
+                'user_first_name': customer.first_name,
+                'user_last_name': customer.last_name,
+                'user_email': customer.email,
+                'product_title': product_title,
+                'product_type': item.product_type.obj.key
+            }
+
+        if is_order_fulfillment_redirection_enabled:
+            logger.info(f'[CT-{tag}] Order Fulfillment Redirection Flag [ENABLED].')
```

Comment:
can we please add other details here like order_id, course_id, user_email so that debugging is easier?